.dockerignore

```diff
@@ -0,0 +1,20 @@
+*.css
+*.html
+*.js
```

Comment:
I don't understand why we need to ignore those, but better keep the Web stuff out of our lawn 😄

Reply:
I simply copied .gitignore and added a few filenames :P